### PR TITLE
blog comparing online and in-person TTT

### PR DESCRIPTION
**Pull request by raynamharris**
Here's a post I wrote. I would appreciate comments/suggestions/edits from @ChristinaLK and @gvwilson

### CODE REVIEW

File: _posts/2016/10/2016-10-22-instructrortraining.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+authors: ["Rayna Harris"]
+title: "A Comparison of Online and In-person Instructor Training Workshops"
+date: 2016-10-22
+time: "16:00:00"
+category: ["Instructor Training"]
+---
+
+I have co-taught three instructor training workshops this year (one online and one in-person with Christina Koch and one online with Greg Wilson.
```
Comment (gvwilson) [inline _posts/2016/10/2016-10-22-instructrortraining.md:10]: missing closing parenthesis

File: _posts/2016/10/2016-10-22-instructrortraining.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+---
+
+I have co-taught three instructor training workshops this year (one online and one in-person with Christina Koch and one online with Greg Wilson. 
+
+## Overview
+Too long to read? Here's a summary in tabular form. 
+
+| Feature | In-person | Online |
+| :--- | :---: | :---: |  
+networking | excellent | poor |
```
Comment (gvwilson) [inline _posts/2016/10/2016-10-22-instructrortraining.md]: Was it poor within each room for the online, or just between the rooms?

File: _posts/2016/10/2016-10-22-instructrortraining.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+| Feature | In-person | Online |
+| :--- | :---: | :---: |  
+networking | excellent | poor | 
+Etherpad use | poor | excellent |
+time commitment | prep + class time + <br> travel + paperwork | prep + class time | 
+private communication | in person | Slack & email |
+technical difficulties | medium | high |
+
+## Networking 
+**In-person:** I think the most amazing feature of an in person workshop is the networking. There is something about face to face conversation during activities and over coffee, beer, and  dinner that really solidifies personal relations. I became a certified instructor in January 2015 at UC Davis TTT taught by Greg Wilson, Tracy Teal, Bill Mills, and Aleksandra Pawlik. I consider all four of these people to be close colleagues now. Also, I'm going to be visiting a handful of the workshop learned in November when I visit California. Also, I talk to a lot of SWC/DC instructors on-line, but there is something about meeting them in person that feels like its the first time we've met.
```
Comment (gvwilson) [inline _posts/2016/10/2016-10-22-instructrortraining.md]: "in-person" (hyphenated)

Comment (gvwilson) [reply _posts/2016/10/2016-10-22-instructrortraining.md]: and "face-to-face" - but maybe that's a Canadian thing

Comment (gvwilson) [reply _posts/2016/10/2016-10-22-instructrortraining.md]: I don't think most people will know what TTT is.

File: _posts/2016/10/2016-10-22-instructrortraining.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+technical difficulties | medium | high |
+
+## Networking 
+**In-person:** I think the most amazing feature of an in person workshop is the networking. There is something about face to face conversation during activities and over coffee, beer, and  dinner that really solidifies personal relations. I became a certified instructor in January 2015 at UC Davis TTT taught by Greg Wilson, Tracy Teal, Bill Mills, and Aleksandra Pawlik. I consider all four of these people to be close colleagues now. Also, I'm going to be visiting a handful of the workshop learned in November when I visit California. Also, I talk to a lot of SWC/DC instructors on-line, but there is something about meeting them in person that feels like its the first time we've met. 
+
+**Online:** The first time I noticed a lack of networking for the online classes was during a debriefing session earlier this week. It wasn't until 45 minutes into the call that I realized that two of the participants were learners in one my my workshops! I'm glad I finally made the connection, but it made me realize who hard it is to recognize people when you only see a tiny image of them on your screen when you are teaching.
```
Comment (gvwilson) [inline _posts/2016/10/2016-10-22-instructrortraining.md]: "my my" => "of my"

Comment (gvwilson) [reply _posts/2016/10/2016-10-22-instructrortraining.md]: "who hard" => "how hard"

File: _posts/2016/10/2016-10-22-instructrortraining.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+**In-person:** During the last in person session with Christina, we often asked students to answer questions out loud rather than using the Etherpad, and one of the comments at the end was that this form meant that a lot of people's thoughts/opinions were never heard. 
+
+**Online:** During the last online session with Greg, we used the Etherpad extensively, so I felt like participation was really high. Also, the extensive note-taking allowed Greg to visualize participation during one of the exercises he had to miss. 
+
+**What does this mean for next time?** I think next time I teach in person, I will use the etherpad chat a lot more, especially for quick yes/no responses.
+
+## Time Commitment
+**In-person:** Even though I really enjoy traveling, saying yes to teaching a workshop in a different cities is a huge time commitment. Instead of just saying yes to teaching from 9-4, I'm saying yes to being in a different city for 24 hours a day. I also have to devote a lot of additional time to planing the travel, traveling, and getting reimbursed for expenses.
```
Comment (gvwilson) [inline _posts/2016/10/2016-10-22-instructrortraining.md]: "a different cities" => "different city"

Comment (gvwilson) [reply _posts/2016/10/2016-10-22-instructrortraining.md]: "planing" => "planning"

File: _posts/2016/10/2016-10-22-instructrortraining.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+**Online:** When I teach online, I really only have to commit time to prep for the lessons and to teach them. I can make breakfast in my own home, eat dinner with friends, and and even make it to meetings and lab meetings that happen on the same day in the same building. 
+
+**What does this mean for next time?** All things considered equal, this factor alone makes me much more willing to say yes to co-teaching online rather than in person workshops.
+
+## Private Communication
+**In-person:** When co-teaching in person, you can easily communicate privately with your co-instructor during the lesson and during the breaks. You never have to worry about wether you are muted or not or if the learners can hear you, and making decisions about wether or not to change the lesson plan on the fly is pretty easy 
+
+**Online:** When co-teaching online, you have to have yet another application open on your computer for private communication. I like Slack a lot for communicating, but it was a little odd when I was screen sharing and some slack notifications came on the screen (yikes!). Email also works, but then I can get distracted by other emails, so this is not ideal.
```
Comment (gvwilson) [inline _posts/2016/10/2016-10-22-instructrortraining.md]: I'd lose the "yikes!"

File: _posts/2016/10/2016-10-22-instructrortraining.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+**Online:** When co-teaching online, you have to have yet another application open on your computer for private communication. I like Slack a lot for communicating, but it was a little odd when I was screen sharing and some slack notifications came on the screen (yikes!). Email also works, but then I can get distracted by other emails, so this is not ideal. 
+
+## Technical Difficulties
+**In-person:** I would rate potential for technical difficulties in the classroom as medium. There's always a chance that the projector system isn't optimal or that the internet connection is poor, but usually an expert in the room can come up with a solution or temporary fix on the fly. 
+
+**Online:** This is a real pain and can eat into your teaching time. I've encountered all sorts of issues including bad bad sound, bad video, poor Etherpad accessibility, inability to screen share, among other things. I don't have an answer for this.
```
Comment (gvwilson) [inline _posts/2016/10/2016-10-22-instructrortraining.md]: "bad bad sound" => "bad sound"

Comment (gvwilson) [reply _posts/2016/10/2016-10-22-instructrortraining.md]: Inconsistent capitalization: "etherpad" vs "Etherpad"

**Comment by jduckles** [comment]
@raynamharris ready to merge?

**Comment by raynamharris** [comment]
I think so.  @jduckles

**Pull request merged**